resources/js/svir_viewer_d3_tree.js

```diff
@@ -666,6 +676,62 @@
                     updateButton(pdId);
                 });
 
+            // Write (ignore weights) in a new line, if present
+            nodeEnter.append("text")
+                .text(function(d) {
+                    if (d.children){
+                        var ignoreWeightsStr = '';
+                        if (d.operator.indexOf('ignore weights') != -1) {
+                            parts = d.operator.split('(');
+                            ignoreWeightsStr = '(' + parts[1];
+                        }
+                        return ignoreWeightsStr;
+                    }
+                })
+                .attr("id", function(d) {return "ignore-weights-label" + d.level;})
+                .attr("x", function(d) { return Math.abs(d.weight) * CIRCLE_SCALE + 15; })
+                .attr("transform", "translate(0, 12)")
+                .style("fill", function(d) {
+                    if (d.operator !== undefined) {
+                        // Check for operators that ignore weights and style accordingly
+                        var color = '#660000';
+                        return color;
+                    }
+                })
+                .on("click", function(d) {
+                    pdName = d.children[0].name;
+                    pdData = data;
+                    pdLevel = d.children[0].level;
+                    pdParent = d.name;
+                    pdParentField = d.field;
+                    pdTempSpinnerIds = [];
+                    pdTempIds = [];
+                    pdOperator = d.operator;
+                    pdId = d.id;
+                    $('#projectDefWeightDialog').empty();
+                    updated_nodes = d.children[0];
+                    findTreeBranchInfo(pdData, [pdName], [pdLevel], pdParentField, updated_nodes);
+                    operatorSelect(pdOperator);
+                    updateButton(pdId);
+                });
+
+            nodeEnter.append("text")
```

Comment:
Maybe add a small comment each time appending text to make note of what is being appended, now that there are many